Bottom: e669b8e92b6a47e0663e1c4be088cbac2ce8a268
Top:    9944c68795ecae5a956612b446fe33e83d80eba8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-15 00:38:52 +0000

Refresh of hid-ps-don-t-use-the-core-s-dr

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 5f262d9..5098dbc 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -571,12 +571,12 @@ ps_expose (void)
   bool empty;
 
   gui = &ps_hid;
-  Output.fgGC = gui->make_gc ();
-  Output.bgGC = gui->make_gc ();
-  Output.pmGC = gui->make_gc ();
+  Output.fgGC = gui->graphics->make_gc ();
+  Output.bgGC = gui->graphics->make_gc ();
+  Output.pmGC = gui->graphics->make_gc ();
 
-  gui->set_color (Output.pmGC, "erase");
-  gui->set_color (Output.bgGC, "drill");
+  gui->graphics->set_color (Output.pmGC, "erase");
+  gui->graphics->set_color (Output.bgGC, "drill");
 
   PCB->Data->SILKLAYER.Color = PCB->ElementColor;
   PCB->Data->BACKSILKLAYER.Color = PCB->InvisibleObjectsColor;
@@ -628,9 +628,9 @@ ps_expose (void)
   if (set_layer ("fab", SL (FAB, 0)))
     PrintFab (Output.fgGC);
 
-  gui->destroy_gc (Output.fgGC);
-  gui->destroy_gc (Output.bgGC);
-  gui->destroy_gc (Output.pmGC);
+  gui->graphics->destroy_gc (Output.fgGC);
+  gui->graphics->destroy_gc (Output.bgGC);
+  gui->graphics->destroy_gc (Output.pmGC);
   gui = old_gui;
 }
